docs/en/Community-Articles/2024-05-07-using-fluent-validation-with-abp/post.md

```diff
@@ -0,0 +1,95 @@
+
+![validation](./images/validation.png)
+
+Validations make your application run more efficiently because they are not connected to the database.  Validations should be applied in different layers of applications. Validations are applied on the UI side. Validations should also be applied on the backend side and database to prevent malicious users from passing these validations. Fluent validation is often used to create validations on the backend side.
+
+## What is Fluent Validation?
+Fluent validation is a library for checking whether data is valid or not. Fluent validation can be applied to your code in a fluent and understandable way.
+
+
+## Why We Should Use Fluent Validation?
+Fluent Validation allows you to define your validation rules in a clear and flexible way. This means you can comfortably handle complex validation scenarios in your code. This makes your development process much more manageable. The readability that Fluent Validation offers really makes things easier. Having a clear understanding of what your validation rules do is a huge advantage when working on your code. In short, your code is cleaner and clearer. Fluent Validation is also very functional in terms of testability. By defining your validation rules in separate classes, you can easily test and maintain these rules. Fluent Validation is a widely used validation library on the .NET platform. As such, it has become a common standard among .NET developers. This provides advantages in terms of community support and compatibility. So, using Fluent Validation simplifies your development process by making your code more understandable, manageable and testable.
+
+## How to Use Fluent Validation in ABP?
+![fluent validation](./images/abp-fluentvalidation.png)
+
+Using Fluent validation with Abp is quite simple. Open a command line window in the folder of the project (.csproj file) and type the following command: 
+abp add-package Volo.Abp.FluentValidation
+If you have not created your abp project, review the [steps to create](https://docs.abp.io/en/abp/latest/Tutorials/Todo/Overall) it now.
+
+Create your product object like this: 
+````csharp
+  public class Product: FullAuditedAggregateRoot<Guid>
+   {
+       public string Name { get; set; } = string.Empty;
+       public decimal Price { get; set; }
+       public int Stock { get; set; }
+       public string? LicenseKey { get; set; }
+    }
+````
+Create the controller of your product object by following the [documentation](https://docs.abp.io/en/abp/latest/Tutorials/Todo/Index?UI=NG&DB=EF) 
```

Comment:
I did not understand this paragraph. Shouldn't you say that "after creating the entity, make the database configurations, create a new migration, and apply it to the database. Then, let's create the DTO classes:"?

If so, please update this line in a more descriptive way, and don't assume that the readers will know the other steps. An article should be descriptive, easy to use and ready to be consumed/used by applying all steps.